### ISSUE TITLE
Support for Rails 6.1

### ISSUE THREAD

**Issue by mscoutermarsh** [cased/cased-rails#1]
Hi 😄 , currently can't install on a Rails 6.1.

```
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 6.1.1)

  In Gemfile:
    rails (= 6.1.1)

    cased-rails was resolved to 0.3.0, which depends on
      rails (~> 6.0.2)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by dewski** [comment]
Hey @mscoutermarsh 👋 thanks for calling this out. We've just released [cased-rails 3.0.1](https://rubygems.org/gems/cased-rails/versions/0.3.1) which relaxes the `rails` version to Rails 6.0 or above.

**Comment by mscoutermarsh** [comment]
🙏 thank you!
